### PR TITLE
[FIX] point_of_sale: verify category sequence in test cases

### PR DESCRIPTION
**Pull request by meji-odoo**
In this commit:
===
- Updated the test case to ensure that when clicking on a parent category, its subcategories are displayed immediately after the parent category.

### CODE REVIEW

**Comment by caburj** [comment]
robodoo fw=no r+

**Comment by meji-odoo** [comment]
@caburj 

I think we need to reapply the r+.

Thanks :)

File: addons/point_of_sale/tests/test_frontend.py [hunk 1]
```diff
@@ -1478,22 +1478,27 @@ def test_product_categories_order(self):
         self.env['pos.category'].create({
```
[the remark this answers is not in the document]
Comment (meji-odoo) [reply addons/point_of_sale/tests/test_frontend.py:1479]: @caburj 
I have updated the code.

Thanks

**Comment by caburj** [comment]
robodoo r+

**Pull request closed without merging**